### DISCUSSION TITLE
Is it possible to use vue-apollo with Nativescript-Vue?

### DISCUSSION THREAD

**Discussion by leopsidom** [Akryum/vue-apollo#611] [Q&A]
As the title, can we use vue-apollo with [nativescript-vue](https://nativescript-vue.org/en/docs/introduction/)? I did some experiments, seems I can't do `vue add apollo` which gives the following error:

 

> ERROR  Error: Cannot find module '@vue/cli-shared-utils'
> Error: Cannot find module '@vue/cli-shared-utils'
>     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
>     at Function.Module._load (internal/modules/cjs/loader.js:508:25)
>     at Module.require (internal/modules/cjs/loader.js:637:17)
>     at require (internal/modules/cjs/helpers.js:22:18)
>     at Object.<anonymous> (C:\Users\Xiaoyun\OnExperiment\native-vue-test\node_modules\vue-cli-plugin-apollo\generator\index.js:4:5)
>     at Module._compile (internal/modules/cjs/loader.js:701:30)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
>     at Module.load (internal/modules/cjs/loader.js:600:32)
>     at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
>     at Function.Module._load (internal/modules/cjs/loader.js:531:3)

**Comment by flieks** [comment]
Also want to know :)

**Comment by Akryum** [comment]
Just as a note: `vue add` only works in `vue-cli` 3+ projects.

**Comment by tafelnl** [comment]
Any updates on this?

Does it have something similar like Angular? https://www.apollographql.com/docs/angular/features/nativescript/
